synapse/tests/test_lib_stormlib_infosec.py

```diff
@@ -0,0 +1,128 @@
+import os
+import asyncio
+
+import synapse.exc as s_exc
+import synapse.common as s_common
+
+import synapse.tests.utils as s_test
+
+res0 = {'ok': True, 'version': '3.1', 'score': None, 'scores': {
+            'base': None, 'temporal': None, 'environmental': None}}
+res1 = {'ok': True, 'version': '3.1', 'score': 10.0, 'scores': {
+            'base': 10.0, 'temporal': None, 'environmental': None, 'impact': 6.0, 'exploitability': 3.9}}
+res2 = {'ok': True, 'version': '3.1', 'score': 10.0, 'scores': {
+            'base': 10.0, 'temporal': 10.0, 'environmental': None, 'impact': 6.0, 'exploitability': 3.9}}
+res3 = {'ok': True, 'version': '3.1', 'score': 9.8, 'scores': {
+            'base': 10.0, 'temporal': 10.0, 'environmental': 9.8, 'impact': 6.0, 'modifiedimpact': 5.9, 'exploitability': 3.9}}
+
+# https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:A/AC:L/PR:H/UI:R/S:C/C:H/I:N/A:L/E:P/RL:T/RC:U/CR:H/IR:L/AR:M/MAV:X/MAC:H/MPR:L/MUI:N/MS:U/MC:H/MI:L/MA:N&version=3.1
+vec4 = 'AV:A/AC:L/PR:H/UI:R/S:C/C:H/I:N/A:L/E:P/RL:T/RC:U/CR:H/IR:L/AR:M/MAV:X/MAC:H/MPR:L/MUI:N/MS:U/MC:H/MI:L/MA:N'
+res4 = {'ok': True, 'version': '3.1', 'score': 5.6, 'scores': {
+            'base': 6.5, 'temporal': 5.4, 'environmental': 5.6, 'impact': 4.7, 'modifiedimpact': 5.5, 'exploitability': 1.2}}
+
```

Comment:
We should be able to punch in the set of example test vectors from https://www.first.org/cvss/v3.1/examples to ensure that we don't have any deviations from the publish examples at a minimum.

Reply:
That list doesn't contain any temporal or environmental score calculations and would require a large transcription effort to get into machine readable form. Not worth it.